literature/MYabrv.bib

```diff
@@ -236,6 +236,7 @@ @String{VLSI
 @String{VR = "Proc.\ IEEE Virtual Reality Workshop (VR)"}
 @String{WASP = "Proc.\ Latin American Workshop on Aspect-Oriented Software Development: Advanced Modularization Techniques (LA-WASP)"}
 @String{WEA = "Proc.\ Int'l Workshop on Experimental and Efficient Algorithms (WEA)"} % now known as SEA
+@String{WSRE = "Proc.\ German Workshop on Software-Reengineering and -Evolution (WSRE)"} %
```

Comment:
No `-` in front of `Evolution`? As far as I know, that is german syntax and does not exist in english.